### ISSUE TITLE
Setting VariantRecord pos or stop raises error

### ISSUE THREAD

**Issue by msto** [pysam-developers/pysam#640]
Hi,

On pysam 0.14, installed via conda, setting the `pos` or `stop` of a VariantRecord raises the following error:

```
Assertion failed: (!inf->vptr_free), function bcf_update_info, file vcf.c, line 3342.
Abort trap: 6
```

I tried looking into what might be causing this, but I'm unable to replicate when installing from source with `pip install -e`. It's only occurring when I try to use the pre-compiled version from bioconda. Any idea what might be causing this? Thanks!

EDIT: the error occurs when building with `python setup.py install` as well

**Comment by mvdbeek** [comment]
I'm seeing the same issue, and using the provided manylinux wheels solves the issue, so maybe that's more or a bioconda problem ?

**Comment by kyleabeauchamp** [comment]
Looks like the problem is discuss upstream as well: https://github.com/samtools/htslib/issues/813

**Comment by jmarshall** [comment]
This has been fixed since pysam 0.16.0 (when used with HTSlib 1.10 — which fixed the underlying problem — or later).
